client/src/lib/error-telemetry.js

```diff
@@ -58,7 +58,15 @@ class ErrorTelemetry {
     });
 
     tail.on('line', data => {
-      outputFunction(this.app, data);
+      logger.debug('Reading line:', data);
+
+      try {
+        const parsedLogData = JSON.parse(data);
```

Comment:
I'm not sure I understand the need for the `parsedLogData` variable since it will only be used once.

Reply:
agreed.

Reply:
Done